File: account/src/main/java/dev/exit/asset/account/AssetAccountRepository.java
package dev.exit.asset.account;

import java.util.List;
import java.util.UUID;

import org.springframework.data.jpa.repository.JpaRepository;

import dev.exit.asset.account.AssetAccount;

public interface AssetAccountRepository extends JpaRepository<AssetAccount, UUID> {

	List<AssetAccount> findByOwnerIdOrderByCreatedAtDesc(String ownerId);
}

File: account/src/main/kotlin/dev/exit/asset/account/AssetAccountRepository.kt
package dev.exit.asset.account

import org.springframework.data.jpa.repository.JpaRepository
import java.util.UUID

interface AssetAccountRepository : JpaRepository<AssetAccount, UUID> {

    fun findByOwnerIdOrderByCreatedAtDesc(ownerId: String): List<AssetAccount>
}
